src/test/java/org/springframework/samples/petclinic/system/ProductionConfigurationTests.java

```diff
@@ -17,6 +17,6 @@
     @Test
     public void testFindAll() throws Exception {
         vets.findAll();
-        vets.findAll(); // served from cache
+        //vets.findAll(); // served from cache
```

Comment:
This can be uncommented in code pulled from this PR downstream. This does not relate to the implementations for milestone 3 but might as well be restored back to how it was initially. I was resolving an issue with the vets table retrieval and forgot to uncomment this.